spec/forms/state_file/esign_declaration_form_spec.rb

```diff
@@ -64,6 +64,43 @@
       end
     end
 
+    context "when has agreed to esign and add pin in maryland" do
```

Comment:
minor rephrasing suggestion: "when the primary and spouse have esigned and added pins in maryland"